src/test/java/seedu/trackermon/model/ModelManagerTest.java

```diff
@@ -1,132 +1,133 @@
-//    @Test
-//    public void hasPerson_personInAddressBook_returnsTrue() {
-//        modelManager.addPerson(ALICE);
-//        assertTrue(modelManager.hasPerson(ALICE));
-//    }
-//
-//    @Test
-//    public void getFilteredPersonList_modifyList_throwsUnsupportedOperationException() {
-//        assertThrows(UnsupportedOperationException.class, () -> modelManager.getFilteredPersonList().remove(0));
-//    }
-//
-//    @Test
-//    public void equals() {
-//        AddressBook addressBook = new AddressBookBuilder().withPerson(ALICE).withPerson(BENSON).build();
-//        AddressBook differentAddressBook = new AddressBook();
-//        UserPrefs userPrefs = new UserPrefs();
-//
-//        // same values -> returns true
-//        modelManager = new ModelManager(addressBook, userPrefs);
-//        ModelManager modelManagerCopy = new ModelManager(addressBook, userPrefs);
-//        assertTrue(modelManager.equals(modelManagerCopy));
-//
-//        // same object -> returns true
-//        assertTrue(modelManager.equals(modelManager));
-//
-//        // null -> returns false
-//        assertFalse(modelManager.equals(null));
-//
-//        // different types -> returns false
-//        assertFalse(modelManager.equals(5));
-//
-//        // different addressBook -> returns false
-//        assertFalse(modelManager.equals(new ModelManager(differentAddressBook, userPrefs)));
-//
-//        // different filteredList -> returns false
-//        String[] keywords = ALICE.getName().fullName.split("\\s+");
-//        modelManager.updateFilteredPersonList(new NameContainsKeywordsPredicate(Arrays.asList(keywords)));
-//        assertFalse(modelManager.equals(new ModelManager(addressBook, userPrefs)));
-//
-//        // resets modelManager to initial state for upcoming tests
-//        modelManager.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
-//
-//        // different userPrefs -> returns false
-//        UserPrefs differentUserPrefs = new UserPrefs();
-//        differentUserPrefs.setAddressBookFilePath(Paths.get("differentFilePath"));
-//        assertFalse(modelManager.equals(new ModelManager(addressBook, differentUserPrefs)));
-//    }
-//}
+package seedu.trackermon.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.trackermon.model.Model.PREDICATE_SHOW_ALL_SHOWS;
+import static seedu.trackermon.testutil.Assert.assertThrows;
+import static seedu.trackermon.testutil.TypicalShows.ALICE_IN_WONDERLAND;
+import static seedu.trackermon.testutil.TypicalShows.HIMYM;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.trackermon.commons.core.GuiSettings;
+import seedu.trackermon.model.show.NameContainsKeywordsPredicate;
+import seedu.trackermon.testutil.ShowListBuilder;
+
+public class ModelManagerTest {
```

Comment:
Please add header comment?

Reply:
A lot of the other test classes do not have a header comment, is someone going to update the all test cases to include a javadoc header?

Reply:
i opened a new issue for this, you can go there and discuss about who is going update what.